drracket-test/tests/drracket/private/drracket-test-util.rkt

```diff
@@ -89,10 +89,7 @@
   (define (wait-for-drracket-frame [print-message? #f])
     (define (wait-for-drracket-frame-pred)
       (define active (fw:test:get-active-top-level-window))
-      (if (and active
-               (drracket-frame? active))
-          active
-          #f))
+      (and (and active (drracket-frame? active)) active))
```

Comment:
we only need one `and` here, but maybe that's out of the scope of this improvement?